potypo/__main__.py

```diff
@@ -43,6 +43,7 @@ def main():
         words = conf['edgecase_words'].strip().split('\n')
         filter_list.append(filters.make_EdgecaseFilter(words))
 
+    wl_dir = conf.get('wl_dir', None)
```

Comment:
I'd move this below the errmsg-function, to have the variable declaration as close to its usage as possible.